### DISCUSSION TITLE
ContainsArrayError: path '' contains an array

### DISCUSSION THREAD

**Discussion by tom221085** [miniscope/minian#150] [Q&A] [answered]
Hello!

For some reason I get the following error message using a data set that I have already successfully analyzed in minian.

![Error message Minian](https://user-images.githubusercontent.com/11939642/123697237-a85faa80-d82a-11eb-9817-a63b84baab3c.jpeg)

Does anyone have an idea why this message pops up?

Thanks,
Thomas

**Comment by tom221085** [comment, marked as answer]
Okay, I think I solved it. I just deleted everything in the minian_intermediate folder. Afterwards, the error was gone.
I'm still not sure what exactly the issue was, though.

**Reply by phildong** [reply]
Hey glad it's solved! Yeah that exception almost always means there's something already at the save path/intermediate path that has a weird folder structure. The exact reason could be hard to track down (but do submit an issue if you're able to come up with a reproducible case!), but deleting the offending folders almost always solve it.

**Comment by hardawayja** [comment]
I also had this error and can confirm that deleting the minian_intermediate folder also got me unstuck. It would still be great to know which variable is causing the error so you don't have to start over. With a large dataset this can take some time to rerun.
